zinc/src/test/scala/sbt/inc/LookupSpec.scala

```diff
@@ -93,17 +67,19 @@ class LookupSpec extends BaseIncCompilerSpec {
     lookup.lookupAnalysis("P1$P2") shouldEqual None
   }
 
-  it should "find correct analysis for class file" in mockedProjectContext { setup =>
-    import setup._
+  // it should "find correct analysis for class file" in mockedProjectContext { setup =>
+  //   import setup._
 
-    lookup.lookupAnalysis(project1.P1P2.classFile) shouldEqual project1.analysis
-    lookup.lookupAnalysis(project2.P1P2.classFile) shouldEqual project2.analysis
-    lookup.lookupAnalysis(project2.P2.classFile) shouldEqual project2.analysis
-    lookup.lookupAnalysis(project1.P1J2.classFile) shouldEqual project1.analysis
-    lookup.lookupAnalysis(project1.P1C2.classFile) shouldEqual project1.analysis
+  //   val lookup = new LookupImpl(config)
```

Comment:
I think we agree. I just changed the test's implementation of Analysis lookup for make sure it can be looked up with class file.